packages/framework/tests/Test/IsMoneyEqual.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\FrameworkBundle\Test;
+
+use PHPUnit\Framework\Constraint\Constraint;
+use Shopsys\FrameworkBundle\Component\Money\Money;
+
+class IsMoneyEqual extends Constraint
```

Comment:
There are missed possible useses:

`BasePriceCalculationTest, PaymentPriceCalculationTest, TransportPriceCalculationTest`

you can find them by regexp `.*True.*get.*->equa`

Reply:
Fixed in 3de2a6564737f1fe4dfbdb0a289c600dcd22e565